src/main/java/no/nav/tag/kontakt/oss/KontaktskjemaController.java

```diff
@@ -24,7 +28,43 @@ public ResponseEntity meldInteresse(
         if (kontaktskjemaService.harMottattForMangeInnsendinger()) {
             return ResponseEntity.status(HttpStatus.TOO_MANY_REQUESTS).build();
         }
+
+        if (!erGyldigKontaktskjema(kontaktskjema)) {
+
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
+        }
+
         kontaktskjemaService.lagreKontaktskjema(kontaktskjema);
         return ResponseEntity.ok(HttpStatus.OK);
     }
+
+    private boolean erGyldigKontaktskjema(Kontaktskjema kontaktskjema) {
+        String latin = "a-zA-Z \\-'";
+        String norsk = "æøåÆØÅ";
+        String sifre = "0-9";
+        String eposttegn = "\\.@+";
+        String aksenter = "ëÿüïöäéúíóáèùìòàêûîôâõãñËŸÜÏÖÄÉÚÍÓÁÈÙÌÒÀÊÛÎÔÂÕÃÑ";
+
+        String rausTekst = latin + norsk + sifre + aksenter;
+        String epost = latin + norsk + sifre + aksenter + eposttegn;
+
+        return erGyldigFelt(kontaktskjema.getBedriftsnavn(), rausTekst) &&
+                erGyldigFelt(kontaktskjema.getFornavn(), rausTekst) &&
+                erGyldigFelt(kontaktskjema.getEtternavn(), rausTekst) &&
+                erGyldigFelt(kontaktskjema.getFylke(), rausTekst) &&
+                erGyldigFelt(kontaktskjema.getKommune(), rausTekst) &&
+                erGyldigFelt(kontaktskjema.getOrgnr(), sifre + " ") &&
+                erGyldigFelt(kontaktskjema.getTelefonnr(), sifre + "+ ") &&
+                erGyldigFelt(kontaktskjema.getEpost(), epost);
+    }
+
+    private boolean erGyldigFelt(String felt, String skalBareInneholde) {
+        Pattern pattern = Pattern.compile("^[" + skalBareInneholde + "]*$");
```

Comment:
Her er det vel også bare tre ulike pattern-varianter? Pattern-kompilering er relativt tungt, så det kan være greit å kompilere patterne på klassenivå og ha dem som konstanter - f.eks. en for rausTekst, en for sifre og en for epost. Mulig koden også blir litt enklere / mer oversiktlig hvis de ligger som prekompilerte Patterns uten at du må sende inn regexen som parameter?